src/main/java/tools/jackson/databind/EnumNamingStrategy.java

```diff
@@ -11,9 +14,13 @@ public interface EnumNamingStrategy {
      * the implementation of this {@link EnumNamingStrategy}.
      *
      * @param enumName the name of the enum value to translate
+     * @param config the mapper configuration
+     * @param cls the annotated class
```

Comment:
Is this the Enum class or something else? (I assume Enum) Should specify, otherwise this has no information

Reply:
True. Will add a note